content/influxdb/cloud-iox/union.md

```diff
@@ -0,0 +1,74 @@
+---
+title: The UNION clause
+list_title: The UNION clause
+description: > 
+    Use the `UNION` clause to combine the results of two or more queries into a single set of results.
+menu:
+  influxdb_cloud_iox:
+    name: The UNION clause
+    parent: Explore data using SQL
+weight: 295
+---
+
+The `UNION` clause combines the results of two or more `SELECT` statements into a single results set.
+
+Key points to keep in mind when using the `UNION` clause:
+
+ - the number of columns in the column list must be the same
+ - the columns in the column list must be in the same order
+ - the data types in the column list must be the same or compatible  
+
+- [Syntax](#syntax)
+- [Examples](#examples)
+
+## Syntax
+
+```sql
+SELECT a, b, c
+FROM measurement_1
+UNION [ALL]
+SELECT a, b, c
+FROM measurement_2
+``` 
+
+## Examples
+
+Basic `UNION` query:
+
+```sql
+SELECT "pH" as "water_pH", "time", "location"
+FROM "h2o_pH"
+UNION 
+SELECT "location", "time", "randtag" 
+FROM "h2o_quality"
+```
+Results:
+
+| location     | time                     | water_ph |
+| :----------- | :----------------------- | :------- |
+| coyote_creek | 2019-08-21T00:00:00.000Z | 8.0      |
+| coyote_creek | 2019-08-21T00:18:00.000Z | 7.0      |
+| coyote_creek | 2019-08-21T00:42:00.000Z | 8.0      |
+
+The query returns the location, time and pH from the 2 tables.
+
+`UNION ALL` query using an `OVER` clause:
+
+```sql
+SELECT 'bottom' as type, time, water_level FROM
+  (SELECT time, "water_level", row_number() OVER (order by water_level) as rn FROM h2o_feet) where rn <= 3 
+UNION ALL 
+SELECT 'top' as type, time, water_level FROM 
+  (SELECT time, "water_level", row_number() OVER (order by water_level DESC) as rn FROM h2o_feet) where rn <= 3
+  ```
```

Comment:
I think this query needs a little more explaining. There isn't really anything special being done with `UNION` here, but the subqueries are fairly advanced and might confuse users.

Reply:
I'm not happy with the dataset and am struggling to come up with UNION queries that make sense.  In retrospect, I would have like to use the sensors data for samples.  But I feel like it's too close to Jan 31 to change now.